packages/amesos2/src/Amesos2_Umfpack_def.hpp

```diff
@@ -250,6 +275,8 @@ Umfpack<Matrix,Vector>::getValidParameters_impl() const
   if( is_null(valid_params) ){
     Teuchos::RCP<Teuchos::ParameterList> pl = Teuchos::parameterList();
 
+    pl->set("IsContiguous", true, "Whether GIDs contiguous");
```

Comment:
nit pick -> "GIDs are contiguous"

Reply:
Good catch, looks like auto-tester already started so I'll clean this up while addressing more issues. Thanks for quick review!!